backend/src/apiserver/resource/resource_manager_test.go

```diff
@@ -231,7 +231,7 @@ func initWithJobV2(t *testing.T) (*FakeClientManager, *ResourceManager, *model.J
 		PipelineSpec: model.PipelineSpec{
 			PipelineSpecManifest: v2SpecHelloWorld,
 			RuntimeConfig: model.RuntimeConfig{
-				Parameters:   "{\"param1\":\"world\"}",
+				Parameters:   "{\"text\":\"world\"}",
```

Comment:
I missed this earlier, but `parameters` belongs to schemaVersion 2.0.0, and is deprecated. Starting from schema 2.1.0, it should be `parameter_values` instead.
https://github.com/kubeflow/pipelines/blob/02ae214223070694624f634e6140a3d0aef92fe5/api/v2alpha1/pipeline_spec.proto#L29-L43

Reply:
Good catch!! I will change accordingly.

Reply:
Since this will be a breaking change for UI and SDK, we won't implement for now.